model/src/main/kotlin/config/LicenseFindingCuration.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model.config
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize
+import com.fasterxml.jackson.databind.annotation.JsonSerialize
+import com.fasterxml.jackson.databind.util.StdConverter
+
+/**
+ * A curation for license finding(s) applying to all license findings for which all matchers represented by this classes
+ * properties match. When applied it assigns the [concludedLicense] to all matching findings.
+ */
+data class LicenseFindingCuration(
+    /**
+     * A glob to match the file path of a license finding.
+     */
+    val path: String,
+    /**
```

Comment:
Add an empty line before the comment, same several times below.

Reply:
Done.